lib/mrn_context_pool.cpp

```diff
@@ -101,6 +101,19 @@ namespace mrn {
       DBUG_VOID_RETURN;
     }
 
+    void set_n_workers(int n_workers) {
+      MRN_DBUG_ENTER_METHOD();
+
+      for (std::vector<grn_ctx *>::iterator it = pool_.begin();
+           it != pool_.end();
+           ++it) {
+        grn_ctx *ctx = *it;
```

Comment:
今ならこれを使えるかも。昔はC++11を使えなかったんだよね。

```suggestion
      for (auto ctx : pool) {
```